src/components/Dex.vue

```diff
@@ -164,6 +170,13 @@ export default {
       });
     },
 
+    loadTickerData() {
+      this.$services.dex.fetchTickerData()
```

Comment:
Why not do this in actions?

Reply:
Good question. I just copied this over from the desktop app. I could move this stuff into actions and just do a dispatch. Is the difference purely just maintaining our pattern, or does this force a re-render or something?